traptor/traptor.py

```diff
@@ -537,7 +565,7 @@ def main(sentry, stdout, info, debug, delay, id, type, key):
         if sentry:
             client = Client(SENTRY_SECRET)
             client.captureException()
-        logger.errror(e)
+        logger.error(e)
```

Comment:
@joequery good question.  It probably does not do anything special with non-ascii characters.  I'm not sure if that situation would arise